### PR TITLE
Updating multicluster support page for `CSI-PowerScale`

### PR DESCRIPTION
**Pull request by rishabhatdell**
# Description
Adding a note for multicluster support in csi-powerscale

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1214 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/features/powerscale.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ You can connect a single CSI-PowerScale driver with multiple PowerScale clusters
 3. Creation of custom-volumesnapshot classes with proper isiPath matching corresponding storage classes.
 4. Inclusion of cluster name in volume handle, if you want to provision existing static volumes.
 
+> The 'clusterName' serves as a logical, unique identifier that should remain unchanged once it is included in the volume handle. Altering this identifier is not advisable, as it would result in the failure of all operations associated with the volume that was created earlier.
```
Comment (adarsh-dell) [inline content/docs/csidriver/features/powerscale.md:18]: How about this: _unique identifier for the array_

**Review by gallacher — CHANGES_REQUESTED** [review]
Missing GitHub issue associated with this chage.

**Review by gallacher — CHANGES_REQUESTED** [review]
Why is this targeted for main and not release-1.11.0?  A known issue for this can be added to main but the doc changes here should be added to release-1.11.0.

**Comment by adarsh-dell** [comment]
Closing this PR, as suggested by @gallacher . This will be part of CSM 1.11.0.

**Pull request closed without merging**